src/substituteteacher.js

```diff
@@ -1,4 +1,6 @@
-(function(window) {
+/* */
+"format cjs";
```

Comment:
Is it important / necessary to specify CommonJS format?  I might just be unaware